rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.spec.ts

```diff
@@ -0,0 +1,293 @@
+import {
+  rundeckServiceMock,
+  rundeckClientMock,
+  axiosMock,
+  i18nMocks,
+  mockEventBus,
+} from "./mock";
+import { mount, VueWrapper } from "@vue/test-utils";
+
+import ActivityList from "../activityList.vue";
+import ActivityFilter from "../activityFilter.vue";
+import OffsetPagination from "../../../../library/components/utils/OffsetPagination.vue";
+
+import { setupRundeckContext } from "./setupRundeckContext";
+import { GlobalOptions } from "./type";
+
+jest.mock("../../../../library/rundeckService", () => rundeckServiceMock);
+jest.mock("@rundeck/client", () => rundeckClientMock);
+jest.mock("axios", () => axiosMock);
+type ActivityListInstance = InstanceType<typeof ActivityList>;
+const defaultProps = {
+  pagination: {
+    total: 10,
+    offset: 0,
+    max: 10,
+  },
+  reports: [],
+  query: {},
+  eventBus: mockEventBus,
+  running: {
+    executions: [],
+  },
+  bulkEditMode: true,
+  bulkSelectedIds: [],
+};
+const mountActivityList = async (
+  props = {},
+  globalOptions: GlobalOptions = {}
+) => {
+  const wrapper = mount(ActivityList, {
+    props: {
+      ...defaultProps,
+      ...props,
+    },
+    global: {
+      components: {
+        OffsetPagination,
+        ActivityFilter,
+      },
+      stubs: {
+        modal: true,
+        "i18n-t": true,
+        ActivityFilter: true,
+        Modal: { template: "<div><slot></slot></div>" },
+        ProgressBar: { template: "<div><slot></slot></div>" },
+        btn: {
+          template: `<button :disabled="disabled" data-test-id="activity-list-delete-selected-executions">Delete Selected Executions</button>`,
+          props: ["disabled"],
+        },
+
+        i18n: true,
+        tooltip: true,
+        ...globalOptions.stubs,
+      },
```

Comment:
2 questions:

- is there any child component that wasn't stubbed or mocked? if not, it's safe to replace the mount with a shallowMount;
- why there are stubs for i18n-t and i18n? these shouldn't be needed, instead, the best approach is to use the global mocks for $t and $tc directives.